### PR TITLE
feat: added nimish to the members list

### PR DESCRIPTION
**Pull request by nimish-gupta**

### CODE REVIEW

File: ids/mapping.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
   "harshith": "harshi7016",
   "manjot": "manjots1607",
   "nikhil": "whydonti",
+  "nimish": "nimish",
```
Comment (Rucha1499) [inline ids/mapping.json:11]: As per mentioned in README.md this is the expected format - ` [member_rds_id]: "<github_id>" `. Kindly replace 
"nimish": "nimish" -> "nimish" : "<your_github_id>". Thank you!

**Review by Rucha1499 — CHANGES_REQUESTED** [review]

**Review by swarajpure — CHANGES_REQUESTED** [review]
>  The face should be visible, centered and identifiable
> Face should cover at least 60% of the image

Thank you!

File: ids/mapping.json (hunk 1 above)
Comment (nimish-gupta) [reply ids/mapping.json:11]: Thanks, changing. Also, will it be better if I update this too?
```
{
	"[member_rds_id]": "[github_id]",
	"first_name": "<first-name>",
	"last_name": "<last-name>",
	"yoe": <years-of-experience>,
	"company": "<company-name>",
	"designation": "<designation>",
	"img": "./img.png",
	"github_id": "<github-id>",
	"linkedin_id": "<linkedin-id>",
	"twitter_id": "<twitter-id>",
	"instagram_id": "<instagram-id>",
	"site": "<bio-website>"
}
```

Comment (Rucha1499) [reply ids/mapping.json:11]:
```{
  "id": "[member_rds_id]",
  "first_name": "Ankush",
  "last_name": "Dharkar",
  "yoe": 8,
  "company": "Pesto Tech",
  "designation": "Instructor",
  "img": "URL to profile pic",
  "github_id": "ankushdharkar",
  "linkedin_id": "ankushdharkar",
  "twitter_id": "ankushdharkar",
  "instagram_id": "ankushdharkar",
  "site": "<bio-website-here>"
}```

This is the object and according to it your details are correct. So, no need of updating it. Also, can you tell from where did you find the object you mentioned above ? Thank you!

Comment (swarajpure) [reply ids/mapping.json:11]: @Rucha1499 I had written that particular object in the step 5 here: https://github.com/Real-Dev-Squad/website-static/blob/main/CONTRIBUTING.md

**Comment by swarajpure** [comment]
Inactive for 2 months

**Pull request closed without merging**